playwright_test/tests/richMessage_Combination_buttons.spec.js

```diff
@@ -45,7 +45,11 @@ let page;
     const iframe = page.frameLocator(widgetLocators.kmIframe)
 
   // The UI of rich message check using snapshot
-    expect(await page.screenshot()).toMatchSnapshot('Button_Combination.png',{threshold: 1});
+    const screenshot = await page.screenshot();
+    expect(screenshot).toMatchSnapshot({
+      threshold: 0.98,
+      name: 'Button_Combination.png'
```

Comment:
move threshold to a constant so that its valued can be tweaked some a common place

Reply:
What do you mean by that ??

Reply:
lets say if we want to change threshold to 0.8 in future, do we need to go through each and every test code and then keep modifying the value?
or there can be a constant defined someone in the common code, modifying that changes it every test case  
https://www.w3schools.com/js/js_const.asp

Reply:
oohhh that constant I was thinking about constant value 
ya ill do that

Reply:
added